packages/aila/src/protocol/schema.ts

```diff
@@ -10,7 +10,9 @@ export const BasedOnSchema = z
       ),
     title: z.string().describe("The human-readable title of the lesson."),
   })
-  .describe("A reference to a lesson plan that this lesson is based on.");
+  .describe(
```

Comment:
This was subtle. But adding this here seemed to fix the issue whereby it would automatically pick a basedOn without user input.